utilities/testing/holohub.container.ctest

```diff
@@ -48,6 +48,7 @@ ctest_read_custom_files("${CTEST_BINARY_DIRECTORY}")
 set(configure_options
   ${CONFIGURE_OPTIONS}
   -D HOLOHUB_DATA_DIR=/workspace/holohub/data-${APP}-${TAG}
+  -D CMAKE_CUDA_COMPILER=/usr/local/cuda/bin/nvcc
```

Comment:
I think we can remove this line, just need the `find_package(holoscan)` above to be kept.